examples/3d/split_screen.rs

```diff
@@ -41,7 +41,16 @@ fn setup(
             ..default()
         },
         cascade_shadow_config: CascadeShadowConfigBuilder {
-            num_cascades: 2,
+            num_cascades: if cfg!(all(
+                feature = "webgl2",
+                target_arch = "wasm32",
+                not(feature = "webgpu")
+            )) {
+                // Currently limited to 1 cascade in WebGL
```

Comment:
```suggestion
                // Limited to 1 cascade in WebGL
```

Small nit, but this is very unlikely to ever change.